client/src/main/java/module-info.java

```diff
@@ -41,5 +41,6 @@
 
 	requires iu.util;
 	requires transitive jakarta.json;
+	requires java.desktop;
```

Comment:
This looks familiar but if it has been explained, I've forgotten already. Is this needed? I didn't see it referenced anywhere other than here.

Reply:
java.desktop includes java.beans.Introspector, which is used by JsonSerializer to find the properties of a Java object. It's odd that something so common for enterprise applications is in the desktop UI development module, but that's where it is so needs to be included. We also use the JavaBeans Introspector for type introspection: see https://github.com/indiana-university/iu-java-util/blob/develop/type/impl/src/main/java/iu/type/TypeTemplate.java, line 285